docs/Settings.md

```diff
@@ -116,7 +116,7 @@ These parameters are accessed by calling [GetParam](./api/GetParam.md) or [SetPa
 |-------------------------------------------|---------------------------|-----------|-----------------------------------------------------------|
 | `QUIC_PARAM_LISTENER_LOCAL_ADDRESS`<br> 0 | QUIC_ADDR                 | Get-only  | Get the full address tuple the server is listening on.    |
 | `QUIC_PARAM_LISTENER_STATS`<br> 1         | QUIC_LISTENER_STATISTICS  | Get-only  | Get statistics specific to this Listener instance.        |
-| `QUIC_PARAM_LISTENER_CID_PREFIX`<br> 2    | uint8_t[]  | Both  | CID prefix prepended to all CIDs.        |
+| `QUIC_PARAM_LISTENER_CIBIR_ID`<br> 2    | QUIC_CIBIR_ID  | Both  | The CIBIR well-known idenfitier.        |
```

Comment:
We should throw this behind preview features if its going to change.

Reply:
Done: https://github.com/microsoft/msquic/pull/2447